app/routes/goal_routes.py

```diff
@@ -0,0 +1,93 @@
+from flask import Blueprint, jsonify, request, make_response, abort
+from app.models.goal import Goal
+from app.models.task import Task
+from app import db
+
+
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+
+@goal_bp.route("", methods=["POST"])
+def create_goal():
+    '''Creates a new Goal record. Title is only required param.'''
+    request_body = request.get_json()
+    try:
+        new_goal = Goal(title=request_body["title"])
+    except KeyError:
+        return {"details": "Invalid data"}, 400
+    
+    db.session.add(new_goal)
+    db.session.commit()
+
+    return {"goal": Goal.goal_response(new_goal)}, 201
```

Comment:
`goal_response` is an instance method, but you're calling it here as a static method.  Python does what you want here, but this is not good practice when working with instances.
```suggestion
    return {"goal": new_goal.goal_response()}, 201
```